modules/vaos/app/services/eps/provider_service.rb

```diff
@@ -2,6 +2,8 @@
 
 module Eps
   class ProviderService < BaseService
+    PAGINATION_TIMEOUT_SECONDS = 45
```

Comment:
Note: This is my paranoia to make sure we do not get stuck in an infinite loop due to bad data.